components/scream/src/physics/p3/tests/p3_ice_supersat_conservation_tests.cpp

```diff
@@ -17,12 +18,15 @@ struct UnitWrap::UnitTest<D>::TestIceSupersatConservation {
 
   static void run_bfb()
   {
+    auto engine = setup_random_test();
+
     IceSupersatConservationData f90_data[max_pack_size];
 
     // Generate random input data
     // Alternatively, you can use the f90_data construtors/initializer lists to hardcode data
     for (auto& d : f90_data) {
-      d.randomize();
+      d.randomize(engine);
+      d.dt = f90_data[0].dt; // hold this fixed, it is not packed data
```

Comment:
When testing calls that take packed data, if those calls also take scalars, those scalars should be held constant.